7b_temp_merge/src/merge_all_temp_dat.R

```diff
@@ -276,27 +276,35 @@ reduce_temp_data <- function(outind, inind) {
 
 }
 
-reduce_reservoir_data <- function(outind, inind) {
+reduce_reservoir_data <- function(outind, drb_ind, nyc_dep_ind) {
   # group by depth category
   # select value closest to noon
-  dat <- readRDS(sc_retrieve(inind)) %>%
+  drb_reservoirs_temps <- readRDS(sc_retrieve(drb_ind)) %>%
     mutate(dateTime_est = as.POSIXct(dateTime, tz = 'America/New_York'),
            time = format(dateTime, '%H:%M'),
            time_hm = lubridate::hm(time),
            hours_minus_noon = time_hm$hour - 12,
            minutes_decimal = time_hm$minute/60,
            hours_diff_noon = abs(hours_minus_noon + minutes_decimal)) %>% arrange(site_id, date, depth_category)
 
-  dat_out <- dat %>%
+  nyc_dep_reservoirs_temps <- readRDS(sc_retrieve(nyc_dep_ind))
+
+  # Selecting unique dates in nyc_dep_reservoirs_temp data.
+  nyc_dep_dates <- unique(nyc_dep_reservoirs_temps$date)
```

Comment:
You have the same issue here as we did before, where you're grabbing dates from both reservoirs, when in reality we want to exclude the dates by reservoir.